### PR TITLE
adding LC 96 fiber patch panel

### PR DESCRIPTION
**Pull request by dejantep**

### CODE REVIEW

File: tests/known-modules.pickle
Comment (DanSheps) [inline tests/known-modules.pickle:1]: Pretty sure we don't want the pickle files included

File: tests/known-slugs.pickle
Comment (DanSheps) [inline tests/known-slugs.pickle:1]: Same as above

**Review by DanSheps — CHANGES_REQUESTED** [review]
Is this simplex 96 or duplex 96?  Seems like a lot of ports.

Also, is it actually generic or is there a specific manufacturer and you are just genericising it?

File: tests/known-modules.pickle
Comment (dejantep) [reply tests/known-modules.pickle:1]: true but i'm not sure what i'm doing wrong

Comment (dejantep) [reply tests/known-modules.pickle:1]: its generic and every port has its own number 1-96 so simplex is possible
its known as LC quad something....

Comment (dejantep) [reply tests/known-modules.pickle:1]: image

**Review by dejantep — COMMENTED** [review]
ok. remove the files

**Review by DanSheps — APPROVED** [review]
Looks good from my end, but I only took a quick look.  I will let @danner26 do a more in-depth dive.  😄

**Comment by dejantep** [comment]
is there anything i can do to help to merge this?

**Comment by danner26** [comment]
Please remove the pickle files from your PR. These should not be included at any time

**Pull request closed without merging**